endpoint_info.go

```diff
@@ -115,6 +116,20 @@ func (epi *endpointInterface) UnmarshalJSON(b []byte) error {
 	return nil
 }
 
+func (epi *endpointInterface) Copy(dstEpi *endpointInterface) error {
+	dstEpi.mac = types.GetMacCopy(epi.mac)
+	dstEpi.addr = types.GetIPNetCopy(epi.addr)
+	dstEpi.addrv6 = types.GetIPNetCopy(epi.addrv6)
+	dstEpi.srcName = epi.srcName
+	dstEpi.dstPrefix = epi.dstPrefix
```

Comment:
You also need to copy the v4PoolID and v6PoolID in order for libnetwork to be able to release the addresses on endpoint delete